.github/workflows/import.yaml

```diff
@@ -54,7 +56,6 @@ jobs:
         run: |
           curl -s \
             -X POST \
-            -H"Authorization: token ${{ secrets.TOKEN_KEYS }}" \
```

Comment:
Took me a while to get back here.

Why is this gone now?

Reply:
Good question. I'm not sure anymore. It indeed looks like it should stay :thinking: